launch_testing/launch_testing/__init__.py

```diff
@@ -0,0 +1,62 @@
+import io
+import re
+from launch.output_handler import LineOutput
+
+
+class InMemoryHandler(LineOutput):
+
+    def __init__(
+        self, name, ld, expected_lines, regex_match=False,
+        filtered_prefixes=[
+            b'pid', b'rc',
+            b'RTI Data Distribution Service Evaluation License issued to',
+            b'Expires on']):
```

Comment:
This requires internal knowledge about the rmw implementations available. For any other rmw implementation it is not possible to extend this. So it's not *dogfooding*. (I don't have a proposal how to handle this atm.)